gui/default/syncthing/core/syncthingController.js

```diff
@@ -1094,7 +1096,11 @@ angular.module('syncthing.core')
             }
 
             // Disconnected
-            return status + 'disconnected';
+            if ($scope.deviceStats[deviceCfg.deviceID].lastSeenDays >= 7) {
+                return status + 'disconnected-inactive';
+            } else {
+                return status + 'disconnected';
+            }
```

Comment:
Just a little nit: I'd check for unused here and not add the `-inactive` suffix then. The two states are the exact same in the web UI and I agree with that semantically: I think unused is the stronger/more relevant state than inactive, and having both doesn't add much IMO.

Reply:
Done.